src/components/IncidenceReportsComponent.jsx

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { Table, Header, Pagination } from 'semantic-ui-react';
+
+import TableRowComponent from './TableRowComponent.jsx';
+import SideMenuComponent from '../_components/SideMenuComponent.jsx';
+
+import { loadIncidenceReports } from '../_actions/incidenceReports.actions';
+
+export class IncidenceReportsComponent extends React.Component {
+  state = {
+    activePage: 1,
+    limit: 10,
+  }
+
+  componentDidMount() {
+    this.props.loadIncidenceReports();
+  }
+
+  handlePaginationChange = (e, { activePage }) => {
+    this.setState({ activePage });
+    this.props.loadIncidenceReports(activePage);
+  }
+
+  handlePageTotal = () => Math.ceil(this.props.incidenceReportsCount / this.state.limit)
+
+  emptyReportsCheck = () => (this.props.reports.length === 0)
+
+  loadTableContent = () => {
```

Comment:
I usually discourage the use of a class method to separate the markup. This tends to build up overtime, making a file too long if future markup additions is done using the same pattern. If you find yourself doing this, then it's time to consider if you need a new component or you can use a ternary operator. Note that if you use a ternary operator and the code looks unreadable, then it means there's a better way of doing it. If in doubt, please come and ask me for clarification.